DeviceAgent.xcodeproj/project.pbxproj

```diff
@@ -45,7 +45,10 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
+		4107F8FD231D7262003961AF /* Resources.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 4107F8FC231D7262003961AF /* Resources.xcassets */; };
```

Comment:
Does the Resources.xcassets need to be included in the  AppStub target?

If there are some unit tests, then I can see why it is a member of the DeviceAgentUnitTest target.

![image](https://user-images.githubusercontent.com/466104/64155086-6f26a100-ce32-11e9-9387-5137b2a9b38d.png)

Reply:
I do not understand the purpose of AppStub, this is why id did not add the asset to this target. I tracked the possible call stack of `[SpringBoardAlerts shared]` and ended up with ```QueryRoutes
+ (NSArray <CBXRoute *> *)getRoutes
```
Do you think it might be a reason to add the asset to the  AppStub target?

I am sure the asset must be added to DeviceAgentUnitTest because there are at least 2 unit tests calling `[SpringBoardAlerts shared]`

Reply:
As for the `DeviceAgentUnitTest`, yep, there are a few tests that covers SpringBoardAlerts class and we need to include resources to DeviceAgentUnitTest target

Reply:
The AppStub is the application that the DeviceAgent XCUITest target is configured to test.  It needs to exist because an XCUITest target (or XCTest target) cannot exist alone.

I am reasonably sure that we should remove AppStub from the target membership and remove the copy-files build phase in the AppStub.

Reply:
i removed resources and dependencies